### DISCUSSION TITLE
The Win10 start menu cannot be opened, and there is no response.

### DISCUSSION THREAD

**Discussion by Oct1AtJoe** [zetaloop/ExplorerPatcher#31] [Q&A] [answered]
The Win10 start menu cannot be opened, and there is no response. The latest version of the software is my system version: 22621.1848. Please give me some guidance

**Comment by zetaloop** [comment]
符号文件是否已经完整下载？
高级设置-打开控制台，重启文件管理器，查看控制台输出的内容。

就系统版本而言，我这儿的 beta 10.0.22621.2050 是可以正常使用 win10 开始菜单的。

**Reply by Oct1AtJoe** [reply]
请问一下 符号文件 在哪下载呢 谢谢您的回复 这是我控制台的内容
![微信图片_20230725143109](https://github.com/zetaloop/ExplorerPatcher/assets/140482860/91ab9a89-ca72-4b21-9857-d7e25502afed)

**Reply by zetaloop** [reply]
符号文件下载在高级设置里打开，默认是开的。
控制台信息有出现符号文件下载的相关信息吗？可能在重启文件管理器后过一点时间再会显示。

**Reply by zetaloop** [reply]
你的软件版本是 22621.1555，系统版本是 22621.1848，可能存在软件作者未适配的问题。也请检查是否有其他软件冲突了。

**Reply by Oct1AtJoe** [reply]
好吧 我再看看吧 谢谢您的回复

**Reply by Oct1AtJoe** [reply]
我仔细查看了符号文件说明还有我的控制台输出日志 在日志上就没有符号文件的相关日志输出 是哪里出了问题吗 我开启着符号文件的下载 请您指点一下

**Reply by zetaloop** [reply]
好神奇啊，我也不知道，回头我有空试试这个版本？

**Reply by Oct1AtJoe** [reply]
好的吧 我再找找原因吧

**Reply by Oct1AtJoe** [reply, marked as answer]
你好 我尝试恢复默认设置然后重新安装了一遍  可以打开win10的开始菜单了 我估计是因为我之前改过注册表的原因 目前已解决感谢回复！

**Reply by joinwuxiao** [reply]
我的也出现了这个问题，开始菜单无法打开。请问如何恢复默认设置？

**Reply by Oct1AtJoe** [reply]
卸载explorerpatcher 打开注册表 搜索explorerpatcher  删除关于它的一切注册表信息 删之前先备份哈
